pkg/crc/config/settings.go

```diff
@@ -91,6 +92,8 @@ func RegisterSettings(cfg *Config) {
 		"Enable experimental features (true/false, default: false)")
 	cfg.AddSetting(EmergencyLogin, false, ValidateBool, SuccessfullyApplied,
 		"Enable emergency login for 'core' user. Password is randomly generated. (true/false, default: false)")
+	cfg.AddSetting(PersistentVolumeSize, constants.DefaultPersistentVolumeSize, validatePersistentDiskSize, SuccessfullyApplied,
+		fmt.Sprintf("Total size in GiB of the persistent volume (must be greater than or equal to '%d')", constants.DefaultPersistentVolumeSize))
```

Comment:
used for what?
it misses explanation as containers arent stored here

Reply:
I am going to use https://github.com/openshift/microshift/blob/main/docs/contributor/rhel4edge_iso.md#disk-partitioning same explanation `Unallocated disk space of 9GB size remains in the rhel volume group to be used by the CSI driver.`  May be change `PersistentVolumeSize` variable to `UnallocatedDiskSize` ?